sdk/redisenterprise/arm-redisenterprisecache/samples-dev/databasesCreateSample.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) Microsoft Corporation.
```

Comment:
@colawwj Could you delete the old samples and re-generated the new samples? Otherwise there will be two duplicated samples with different formats